platform/security/idp/security-idp-client/src/main/java/org/codice/ddf/security/idp/client/IdpHandler.java

```diff
@@ -329,7 +329,7 @@ private HandlerResult doPaosRequest(ServletRequest request, ServletResponse resp
             }
             authnRequest = createAndSignAuthnRequest(true,
                     wantSigned && idpssoDescriptor.getWantAuthnRequestsSigned());
-            paosRequest = createPaosRequest();
+            paosRequest = createPaosRequest((HttpServletRequest) request);
```

Comment:
Hm... I'm not sure. There are some ServletRequest wrapper classes that are concrete and not of the HttpServletRequest type. I think literally everything else extends the HttpServletRequest class. We do make the assumption up at the beginning of the getNormalizedToken method that everything is HTTP though. If we wanted to fix this, I think we'd need to go through all of our handlers and filters everywhere. This is probably fine for now, but maybe one of those tech debt-ish type things.